packages/core-ui/lib/core/index.tsx

```diff
@@ -267,9 +267,7 @@ export const Tx = ({
   name,
   notEnoughFunds,
   dangerMessage,
-  requiresManualSign,
-  ManualSign,
-  DefaultSign,
+  sign,
```

Comment:
Component props should be capitalised, which is why i suggested the name `SignerComponent` in the dashboard PR.